contrib/utxo-tools/utxo_to_sqlite.py

```diff
@@ -16,6 +16,11 @@
 import sys
 import time
 
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '../../test/functional'))
```

Comment:
this is a bit hacky, but I don't mind if we can dedup

Reply:
+1, I felt the same but I feel it's worth getting rid of duplication.

Reply:
Though this means this utility file isn't as self-contained anymore (i.e. you can't just copy it out of the project - not even sure that was possible before)